intermediate_oop/tic_tac_toe/js/main.js

```diff
@@ -1,2 +1,47 @@
 document.addEventListener("DOMContentLoaded", function() {
+	let game = new Grid();
+	let $square = $(".square-set");
+	let $result = $(".result");
+	let $reset = $("#new-game")
+
+	//Move
+	$square.on("click",function(event) {
+		if($(event.target).text() === "" && game.on) {
+			if (game.move % 2) {
+				$(event.target).text("O");
+				game.addResult("O",$(event.target)[0].id);
```

Comment:
careful about mixing your jQuery with vanilla JS. In this case, $(event.target).attr('id') should work.